src/native-stack/views/HeaderConfig.tsx

```diff
@@ -1,107 +1,99 @@
+import { Route, useTheme } from '@react-navigation/native';
 import * as React from 'react';
 import {
   ScreenStackHeaderBackButtonImage,
+  ScreenStackHeaderCenterView,
   ScreenStackHeaderConfig,
-  ScreenStackHeaderRightView,
   ScreenStackHeaderLeftView,
-  ScreenStackHeaderCenterView,
+  ScreenStackHeaderRightView,
 } from 'react-native-screens';
-import { Route, useTheme } from '@react-navigation/native';
 import { NativeStackNavigationOptions } from '../types';
 
 type Props = NativeStackNavigationOptions & {
   route: Route<string>;
 };
 
-export default function HeaderConfig(props: Props) {
+const HeaderConfig: React.FC<Props> = ({
```

Comment:
Same as above

Reply:
Same reasoning as why the other was changed. I also feel it's cleaner to destructure the props from inside the `()` rather than stating props and then stating props again along with destructuring them - it can be done in place of.

Reply:
I agree that we can destructure the props inside the (), but we should keep it as a function.